components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -728,6 +728,10 @@ class RewardsServiceImpl : public RewardsService,
     const std::vector<std::string>& id_list,
     ledger::ResultCallback callback) override;
 
+  void DeleteUnblindedTokensForPromotion(
+    const std::string& id_list,
```

Comment:
should be `promotion_id`

Reply:
Wow - that's a bad oversight - good spot, thank you.

Reply:
Fixed.